### PR TITLE
Feature/fix switching tvdb show language

### PR DESCRIPTION
**Pull request by p0psicles**
- [x] PR is based on the DEVELOP branch
- [x] Don't send big changes all at once. Split up big PRs into multiple smaller PRs that are easier to manage and review
- [x] Read [contribution guide](https://github.com/PyMedusa/SickRage/blob/master/.github/CONTRIBUTING.md)
- [x] Prevent the db from being updated to the new language, if that doesn't have any results.

### CODE REVIEW

**Comment by p0psicles** [comment]
@duramato Is this what you had in mind? I'm stopping the show refresh, and giving some feedback through the ui.notifications(). Don't really know, how to handle it any other better way.

File: medusa/server/web/home/handler.py [hunk 1]
```diff
@@ -1222,6 +1222,27 @@ def sceneExceptions(show):
...
+        If the indexer throws the ShowNotFoundInLanguage Exception, we catch it,
+        and report it's not available in this language.
+        :param show_obj: (TVShow) Show object.
+        :param language: Language passed as string as a two letter country ccde. For ex: 'en'.
+
+        :returns: True (show found in language) False (show not found in language)
+        """
+        indexer_api_params = app.indexerApi(show_obj.indexer).api_params.copy()
+        indexer_api_params['language'] = language
+        indexer_api_params['episodes'] = False
```
Comment (p0psicles) [inline medusa/server/web/home/handler.py]: To give you guys a small update. I've changed the way the episode download is triggered. Previously it was passed as a param, but always passed as True. Meaning for every show search, you had no choice other then to always also download the episodes. Now the param config['episodes_enabled'] can be set, to toggle it.

**Comment by fernandog** [comment]
@p0psicles 

message is that long by design?  `2016-11-11 09:46:30 WARNING  SHOWQUEUE-UPDATE :: [2dd7161] 282670: Data retrieved from TVDBv2 was incomplete. The indexer does not provide show information in the searched language pt. Aborting: Missing attribute series_name, cant index in language: pt` (edited)

i miss the seriesname in the message. ofc i can use the `282670`

**Comment by p0psicles** [comment]
There can be multiple reasons why it had to abort. So some parts are concatenated. Of course we add the show name or indexerid.

**Pull request merged**